cli/src/main/java/com/box/l10n/mojito/cli/command/checks/AbstractCliChecker.java

```diff
@@ -19,6 +19,8 @@ public abstract class AbstractCliChecker {
 
   protected CliCheckerOptions cliCheckerOptions;
 
+  public static final String BULLET_POINT = "\t• ";
```

Comment:
So the problem is that spaces at the beginning are being ignored/removed on Github. I am using tab now to see if it works, but I am not sure if will work as expected. 

![image](https://github.com/box/mojito/assets/44348924/35acb60b-70fd-4f66-9648-45214e01e685)